app/controllers/items_controller.rb

```diff
@@ -11,9 +11,9 @@ def new
   end
 
   def create
-    @presenter = NewItemPresenter.new(Item.new(item_params))
+    @presenter = NewItemPresenter.new(current_user.listings.new(item_params))
```

Comment:
This is an unnecessary coupling to the User model association. The only thing that this is buying us is that it adds the seller_id to the model at the price of us relying on an association in the user model. I would recommend avoiding this type of couplings whenever possible.